indico/modules/events/timetable/operations.py

```diff
@@ -51,6 +51,13 @@ def create_break_entry(event, data):
     return create_timetable_entry(event, entry_data)
 
 
+def create_session_block_entry(session_, data):
```

Comment:
Is this really better than calling `create_session_block()` from the RH and then `create_timetable_entry()`?